pkgs/by-name/pa/paq/package.nix

```
@@ -0,0 +1,26 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "paq";
+  version = "1.1.1";
+
+  src = fetchFromGitHub {
+    owner = "gregl83";
+    repo = "paq";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-oB805M37oLLV2Nttchwzd6V2AUqCo8JFhj6Mfg/JfH0=";
+  };
+
+  cargoHash = "sha256-ziYcaUGYxp+gR5v/yxQElNGLugo3bJtjVwCaHiFkMpw=";
+
+  meta = {
+    description = "Hash file or directory recursively";
+    homepage = "https://github.com/gregl83/paq";
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [ lafrenierejm ];
+  };
```

```suggestion
    mainProgram = "paq";
  };
```
the only executable must be set as `meta.mainProgram`

---

Good catch! Fixed just now in 434eea897504dc529af2617f8f072b774f7e2ce0.